cmd/thanosconvert/main.go

```diff
@@ -0,0 +1,88 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/go-kit/kit/log/level"
+	"github.com/weaveworks/common/logging"
+	"gopkg.in/yaml.v2"
+
+	"github.com/cortexproject/cortex/pkg/storage/bucket"
+	"github.com/cortexproject/cortex/pkg/util/log"
+	"github.com/cortexproject/cortex/tools/thanosconvert"
+)
+
+var (
+	configFilename string
+	dryRun         bool
+)
+
+func main() {
+
+	logfmt, loglvl := logging.Format{}, logging.Level{}
+	logfmt.RegisterFlags(flag.CommandLine)
+	loglvl.RegisterFlags(flag.CommandLine)
+	flag.StringVar(&configFilename, "config", "", "Path to bucket config YAML")
```

Comment:
Nit: I'd suggest supporting command line options too (or perhaps as the only way of configuring). Bucket config needs just a couple of options.